src/errors.cpp

```diff
@@ -102,13 +96,11 @@ class dispatch_category_t:
         if(code == cocaine::error::dispatch_errors::uncaught_error)
             return "uncaught invocation exception";
 
-        return "cocaine.rpc.dispatch error";
+        return cocaine::format("generic %s error", name());
```

Comment:
It does, but for known errors. For unknown errors it shows nothing, because you eat those information.